gotext_test.go

```diff
@@ -178,29 +178,29 @@ msgstr "Another text on another domain"
 	}
 
 	// Test IsTranslation functions
-	if !IsTranslated("My text") {
+	if !IsTranslated("My text", "en_US") {
```

Comment:
After making `lang` param optional, this tests will have to cover both use cases (with and without param).